src/components/OSCALCatalogGroups.js

```diff
@@ -52,32 +53,69 @@ const ComponentTab = styled(Tab)(({ theme }) => ({
   textTransform: "none",
 }));
 
-function a11yProps(index, title) {
+function a11yProps(groupId) {
```

Comment:
Why do we no longer use the title?

Reply:
My concern here is that while `group.title` _is_ required, `group.id` is not. Do we still provide enough information to be useful when the ID is not provided?

Reply:
Looking back at the spec now, I'm realizing `group.title` is the only required field (`[1]`) underneath `group`. Ideally `group.id` will be handled here, but I think the `group.title` will be an okay substitute when the `group.id` isn't provided. I will have to tweak this case at the `OSCALCatlogGroups` level in order to work properly with fragments.

Reply:
This should now function as stated above. If `group.id` is not provided, `group.title will` now act as an identifier (and is formatted with hyphens and lowercased). For example, say the Awareness and Training control group for a SP500-83 catalog doesn't have a group.id: instead it's title (Awareness and Training) is used and the fragment becomes `awareness-and-training/at-2/at-2.1` when clicking the "at-2.1" sub-control anchor link.